### PR TITLE
fix(curriculum): test in password generator project, step 54

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #54647

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-regular-expressions-by-building-a-password-generator/6564f834dd717998092cfd47.md [hunk 1]
```diff
@@ -14,7 +14,13 @@ After your new comment, write a `for` loop to iterate over the `constraints` lis
 You should write a `for` loop to iterate over the `constraints` list with `constraint` and `pattern`.
```
Comment (zairahira) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-regular-expressions-by-building-a-password-generator/6564f834dd717998092cfd47.md:14]: Both hints seem quite similar. The first one can be updated as:

```suggestion
You should write a `for` loop that uses the `constraints` list as the loop variable.
```

**Review by zairahira — COMMENTED** [review]
The test is now checking for `pass` to resolve the console exception👍️
The suggested change in hint isn't compulsory.

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-regular-expressions-by-building-a-password-generator/6564f834dd717998092cfd47.md (hunk 1 above)
Comment (Dario-DC) [reply curriculum/challenges/english/07-scientific-computing-with-python/learn-regular-expressions-by-building-a-password-generator/6564f834dd717998092cfd47.md]: I forgot to change the first after adding the second one.

**Review by zairahira — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Pull request merged**